alpha-lab/phone-neon/index.md

```diff
@@ -1,118 +1,103 @@
 ---
 title: Uncover gaze behaviour on phone screens with Neon
 description: "Evaluate Neon's accuracy on phone screens"
-permalink: /alpha-lab/phone-screens
+permalink: /alpha-lab/phone-neon
+meta:
+  - name: twitter:card
+    content: player
+  - name: twitter:image
+    content: "https://i.ytimg.com/vi/gp5O1uskDME/maxresdefault.jpg"
+  - name: twitter:player
+    content: "https://www.youtube.com/embed/gp5O1uskDME"
+  - name: twitter:width
+    content: "1280"
+  - name: twitter:height
+    content: "720"
+  - property: og:image
+    content: "https://i.ytimg.com/vi/gp5O1uskDME/maxresdefault.jpg"
 tags: [Neon, Cloud]
 ---
 
 <script setup>
 import TagLinks from '@components/TagLinks.vue'
 </script>
 
-# Uncover gaze behaviour on phone screens with Neon
+# Uncover Gaze Behaviour on Phone Screens With Neon
 
 <TagLinks :tags="$frontmatter.tags" />
 
 <Youtube src="gp5O1uskDME"/>
 
-Have you ever wondered what your eyes focus on when scrolling through your favourite app?
+What catches your attention and how do you visually navigate through the interface/page of a mobile app? This article shows you the tools you can use to address these questions. 
 
-In this article, we test whether Neon, our latest eye tracker, can accurately capture and characterise viewing behaviour
-while gazing at small icons on mobile applications. To achieve this, we used some of Alpha Lab's existing tutorials, including
-how to generate scanpaths, define areas of interest (AOIs) and calculate outcome metrics, and map gaze onto dynamic content.
+## Gaze Behaviour During Mobile App Engagement
 
-Shall we start?
+Researchers in fields such as UI/UX and neuromarketing frequently investigate user visual engagement with mobile applications. Wearable eye tracking is sometimes used for this purpose as it can provide valuable insights into user experience during mobile browsing. In this article, we used our latest wearable eye tracker, Neon, to capture and analyse users' viewing behaviour when focusing on small icons and features of mobile applications.
 
-## What you'll need
+## Overcoming Usability Challenges With Neon
 
-Below you can find the tools we used for this project. Using these, you can replicate the content of this article with
-your own applications.
```

Comment:
This section seems a little awkward IMO. I would consider omitting it or just condensing into a single sentence and putting into the previous section - `Gaze Behaviour During Mobile App Engagement`